File: 2024206/lswmission01.c
#include <stdio.h>
#include <stdlib.h>
#include <wiringPi.h>
#include <unistd.h>
#include <time.h>

#define pin_RED 7
#define pin_GREEN 21
#define pin_BLUE 22

int main() {
    wiringPiSetup();
    
    pinMode(pin_RED, OUTPUT);
    pinMode(pin_GREEN, OUTPUT);
    pinMode(pin_BLUE, OUTPUT);

    for (int i = 0; i < 10; ++i) {
        digitalWrite(pin_RED, HIGH);
        usleep(500000);
        digitalWrite(pin_RED, LOW);
        usleep(500000);

        digitalWrite(pin_GREEN, HIGH);
        usleep(500000);
        digitalWrite(pin_GREEN, LOW);
        usleep(500000);

        digitalWrite(pin_BLUE, HIGH);
        usleep(500000);
        digitalWrite(pin_BLUE, LOW);
        usleep(500000);
    }

    return 0;
}

File: 2024206/lswmission01.py
import RPi.GPIO as GPIO
import time

pin_RED = 4
pin_GREEN = 5
pin_BLUE = 6

GPIO.setmode(GPIO.BCM)
GPIO.setup(pin_RED, GPIO.OUT)
GPIO.setup(pin_GREEN, GPIO.OUT)
GPIO.setup(pin_BLUE, GPIO.OUT)

i = 0

for i in range(10):
	GPIO.output(pin_RED, 1)
	time.sleep(0.5)
	GPIO.output(pin_RED, 0)
	time.sleep(0.5)
	GPIO.output(pin_GREEN, 1)
	time.sleep(0.5)
	GPIO.output(pin_GREEN, 0)
	time.sleep(0.5)
	GPIO.output(pin_BLUE, 1)
	time.sleep(0.5)
	GPIO.output(pin_BLUE, 0)
	time.sleep(0.5)


GPIO.cleanup()


